### PR TITLE
RFE: golang: add missing architectures

### PR DESCRIPTION
**Pull request by kolyshkin**
Add a few more missing architectures:
 - LOONG64
 - M68K
 - SH and SHEB

### CODE REVIEW

**Comment by zhaixiaojuan** [comment]
@kolyshkin 
Hi kolyshkin，
I'm very sorry, maybe my understanding is wrong, it is mentioned in this PR that ArchLOONGARCH64 should be used. In the libseccomp-go project, how do you understand this problem.

**Comment by kolyshkin** [comment]
> @kolyshkin Hi kolyshkin， I'm very sorry, maybe my understanding is wrong, it is mentioned in this PR that ArchLOONGARCH64 should be used. In the libseccomp-go project, how do you understand this problem.

Reworked to put the "ARCH" suffix back. It is now LOONGARCH64 everywhere.

**Comment by AkihiroSuda** [comment]
What's current status?

File: seccomp.go [hunk 1]
```diff
@@ -171,6 +171,8 @@ const (
 	ArchPARISC64
 	// ArchRISCV64 represents RISCV64
 	ArchRISCV64
+	// ArchLOONGARCH64 represents 64-bit LoongArch.
+	ArchLOONGARCH64
```
Comment (AkihiroSuda) [inline seccomp.go:175]: `ArchM68K`, `ArchSH`, and `ArchSHEB` have to be added as well

**Comment by zhaixiaojuan** [comment]
@kolyshkin 
Hi kolyshkin, Could you consider merging this PR?

**Comment by znley** [comment]
What's current status?

File: seccomp.go (hunk 1 above)
Comment (pcmoore) [reply seccomp.go:175]: Marking this as resolved in the latest revision.

**Comment by pcmoore** [comment]
Looks good to me.
```
Acked-by: Paul Moore <paul@paul-moore.com>
```

**Review by drakenclimber — APPROVED** [review]
Looks good to me.  Thanks, @kolyshkin 

`Acked-by: Tom Hromatka <tom.hromatka@oracle.com>`

**Review by timscharfenort8 — APPROVED** [review]
good

**Review by pcmoore — APPROVED** [review]

**Pull request merged**